docs/documentation/adding-css-javascript-and-images.md

```diff
@@ -9,11 +9,18 @@ The Prototype Kit processes all the files in the `/app/assets` folder, and puts
 
 Do not change files in the `/public` folder because it’s deleted and rebuilt every time you make a change to your prototype.
 
-## CSS
+Learn how to add:
 
-CSS lets you change how web pages look, for example text sizes, colours or spacing.
+- [CSS](#add-css)
+- [JavaScript](#add-javascript)
+- [images](#add-images)
+- [other files](#add-other-files)
```

Comment:
Are we 100% sure we want to use anchor links? Asking because of the [service manual guidance](https://www.gov.uk/guidance/content-design/links#avoid-anchor-links).

Reply:
@joelanman Thanks for researching into anchor links! Can see from the Slack convo on the Design channel that we're ok to use them here. I'll close this comment thread.